internal/cmd/commands/targetscmd/ssh_funcs.go

```diff
@@ -151,6 +158,14 @@ func extraSshFlagsHandlingFuncImpl(c *SshCommand, _ *base.FlagSets, opts *[]targ
 		*opts = append(*opts, targets.WithWorkerFilter(c.flagWorkerFilter))
 	}
 
+	switch c.flagAddress {
+	case "":
+	case "null":
+		*opts = append(*opts, targets.DefaultAddress())
```

Comment:
I thought we discussed not having this magic `"null"` and instead have the setter and remover calls? If this is the `target create` endpoint, I would expect any presence of the address flag to be a valid address. Maybe they want to connect to a host with a hostname of `null`?

Reply:
We also mentioned that any address we accept when creating a Host should be what we accept here. Currently, [we don't accept a `null` host address](https://github.com/hashicorp/boundary/blob/main/internal/cmd/commands/hostscmd/static_funcs.go#L77-L83), so this maintains consistency from a CLI perspective.

Reply:
That's fair, though I hate it :P